torchvision/csrc/cpu/ROIPool_cpu.cpp

```diff
@@ -123,9 +123,9 @@ void RoIPoolBackward(
 std::tuple<at::Tensor, at::Tensor> ROIPool_forward_cpu(
     const at::Tensor& input,
     const at::Tensor& rois,
-    const float spatial_scale,
-    const int pooled_height,
-    const int pooled_width) {
+    const double spatial_scale,
```

Comment:
I'm fairly confident the change from float to double is the right call here, but I would like a second opinion on whether there is any unforeseen consequence due to the [template](https://github.com/pytorch/vision/blob/bfe9bbc65c6fabb48ab200a7e8dab5be5591a471/torchvision/csrc/cpu/ROIPool_cpu.cpp#L14) definition of `RoIPoolForward(`.

cc @fmassa @mthrok